MODULE.bazel

```diff
@@ -1,2 +1,23 @@
 # TODO: migrate all dependencies from WORKSPACE to MODULE.bazel
 # https://github.com/protocolbuffers/protobuf/issues/14313
+module(
+    name = "protobuf",
+    compatibility_level = 1,
+    version = "27.0",
+)
+
+# LOWER BOUND dependency versions.
```

Comment:
Is it possible to set upper bounds too or pin at a specific version if necessary?